rainier-core/src/main/scala/com/stripe/rainier/core/Discrete.scala

```diff
@@ -2,8 +2,14 @@ package com.stripe.rainier.core
 
 import com.stripe.rainier.compute.{If, Real}
 
-trait Discrete extends Distribution[Int] {
+trait Discrete extends Distribution[Int] { self: Discrete =>
   def logDensity(v: Real): Real
+
+  def zeroInflated(psi: Real) =
+    DiscreteMixture(Map(DiscreteConstant(0.0) -> (1 - psi), self -> psi))
+
+  def constantInflated(constant: Real, psi: Real) =
+    DiscreteMixture(Map(DiscreteConstant(constant) -> (1 - psi), self -> psi))
```

Comment:
Is this backwards? I would expect `psi` to be the probability of the constant.

Reply:
Ah, for consistency with Lambert ,et. al., you're right. I was thinking, "the probability of non-zero" when I wrote it. And that the expression for the expected value was easier `psi*meanOfTheNonZeroDist`. It will turn into `(1-psi)*meanOfTheNonZeroDist`.

Reply:
Even apart from consistency with anything external, I think it's just much more intuitive if when you're calling `constantInflated(k, psi)`, the `k` and `psi` are directly related. So I'd rather we switched it to that.